R/Reactant/build_tarballs.jl

```diff
@@ -27,32 +30,58 @@ if [[ "${bb_full_target}" == x86_64-apple-darwin* ]]; then
     popd
 fi
 
-apk add bazel --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing/
 apk add py3-numpy py3-numpy-dev
```

Comment:
Current error seems to be that `zlib` is not installed.

```suggestion
apk add py3-numpy py3-numpy-dev zlib
```